concepts/numbers/about.md

```diff
@@ -55,6 +55,75 @@ There are three types of maximum (and minimum / maximum negative) values for num
 
 Because of how numbers in JavaScript are implemented, **not** every number between `Number.MIN_VALUE` and `Number.MAX_VALUE` can be represented.
 However, _every_ number between `Number.MIN_SAFE_INTEGER - 1` and `Number.MAX_SAFE_INTEGER + 1` **can** be represented.
+## Infinity - Positive and Negative 
+
+The default value of [`infinity`][infinity] is greater than any other number, by default it's set to `Number.POSITIVE_INFINITY`.
+Here you can see how `Infinity` can be used : 
+```javascript
+console.log(Infinity          ); /* Infinity */
+console.log(Infinity + 1      ); /* Infinity */
+console.log(Math.pow(10, 1000)); /* Infinity */
+console.log(Math.log(0)       ); /* -Infinity */
+console.log(1 / Infinity      ); /* 0 */
+console.log(1 / 0             ); /* Infinity */
+```
+Beware that [`positive-infinity`][positive-infinity] behaves differently than the mathematical infinity. So you may encounter some unexpected behaviors like this:
+- `POSITIVE_INFINITY`, divided by either `NEGATIVE_INFINITY` or `POSITIVE_INFINITY`, is `NaN`.
+- `POSITIVE_INFINITY`, divided by any negative value except `NEGATIVE_INFINITY`, is `NEGATIVE_INFINITY`.
+- Zero multiplied by `POSITIVE_INFINITY` is `NaN`.
+
+[`negative-infinity`][negative-infinity] is the same as `POSITIVE_INFINITY` but with a negative sign :
+- `NEGATIVE_INFINITY`, divided by either `NEGATIVE_INFINITY` or `POSITIVE_INFINITY`, is `NaN`.
+- `NEGATIVE_INFINITY`, divided by any negative value except `NEGATIVE_INFINITY`, is `POSITIVE_INFINITY`.
+- Any positive value, including `POSITIVE_INFINITY`, multiplied by `NEGATIVE_INFINITY` is `NEGATIVE_INFINITY`.
+
+
+## NaN or Not-a-Number
+
+The [`NaN`][NaN] is a variable in global scope. There are five different types of operations returning `NaN`:
+
+-  Number cannot be parsed (e.g. parseInt("blabla") or Number(undefined))
+-  Math operation where the result is not a real number (e.g. Math.sqrt(-1))
+-  Operand of an argument is NaN (e.g. 7 ** NaN)
+-  Indeterminate form (e.g. 0 * Infinity, or undefined + undefined)
+-  Any operation that involves a string and is not an addition operation (e.g. "foo" / 3)
```

Comment:
This content is a 1to1 copy from MDN. If you copy content from external sources you need to do so as a quote and mention the source. You can see an example how to do this here:
https://github.com/exercism/javascript/blob/main/concepts/array-analysis/introduction.md?plain=1#L33
https://github.com/exercism/javascript/blob/main/concepts/array-analysis/introduction.md?plain=1#L85

This comment applies to all the content you added. If you copied content, make sure to either rewrite things in your own words or use quotes with sources.

Reply:
I shall update this accordingly